bin/node/src/service.rs

```diff
@@ -200,7 +200,7 @@ fn setup(
         keystore: keystore_container.sync_keystore(),
         task_manager,
         transaction_pool,
-        rpc_extensions_builder,
+        rpc_builder: rpc_extensions_builder,
```

Comment:
Nit: rename the variable to avoid the `: `.

Reply:
Done